Apps/ClinicalDocument/src/Services/ClinicalDocumentService.cs

```diff
@@ -67,13 +73,14 @@ public async Task<RequestResult<IEnumerable<ClinicalDocumentRecord>>> GetRecords
             if (response.ResultStatus == ResultType.Success)
             {
                 this.logger.LogDebug("PID Fetched: {Pid}", response.ResourcePayload?.PatientIdentity?.Pid);
-                IApiResponse<IEnumerable<ClinicalDocumentRecord>> apiResponse =
-                    await this.clinicalDocumentsApi.GetClinicalDocumentRecordsByPid(response.ResourcePayload?.PatientIdentity?.Pid.ToString()).ConfigureAwait(true);
+                IApiResponse<PhsaHealthDataResponse> apiResponse =
+                    await this.clinicalDocumentsApi.GetClinicalDocumentRecords(response.ResourcePayload?.PatientIdentity?.Pid.ToString()).ConfigureAwait(true);
                 if (apiResponse.IsSuccessStatusCode)
                 {
+                    IList<ClinicalDocumentRecord> clinicalDocuments = this.autoMapper.Map<IList<ClinicalDocumentRecord>>(apiResponse.Content?.Data);
                     requestResult.ResultStatus = ResultType.Success;
-                    requestResult.ResourcePayload = apiResponse.Content;
-                    requestResult.TotalResultCount = apiResponse.Content.Count();
+                    requestResult.ResourcePayload = clinicalDocuments;
+                    requestResult.TotalResultCount = apiResponse.Content?.Data.Count();
```

Comment:
Nit: using `clinicalDocuments.Count` here would remove the need for the null-conditional operator.

Reply:
Good point, I will fix that with my unit tests.